### ISSUE TITLE
Display grant in Study pane

### ISSUE THREAD

**Issue by kelshmo** [Sage-Bionetworks/dccmonitor#70]
Can we add the grant annotation to display here (maybe next to assay or modifiedOn)? We will be able to more easily match this back to grant milestones at a glance!
![Screen Shot 2020-03-05 at 11 35 07 AM](https://user-images.githubusercontent.com/40647130/76022359-d275b680-5edb-11ea-85ea-b2b517d87018.png)

**Comment by Aryllen** [comment]
Sure!

**Comment by Aryllen** [comment]
I spoke too soon, haha. We don't annotate with the grant in dccvalidator so dccmonitor doesn't get that information. The only way to get that information is if the person uploading a metadata file has added that as a column. There could potentially be a function that checks the file data for the grant and then updates the table, but the contributor may not have put that column in. I'll think more on this.

**Comment by karawoo** [comment]
> There could potentially be a function that checks the file data for the grant and then updates the table, but the contributor may not have put that column in. 

It's required to be present and complete in the manifest for both AMP-AD and PEC, which helps somewhat, but of course that doesn't mean it is there, just that the contributor will see failures if it isn't.

**Comment by Aryllen** [comment]
> It's required to be present and complete in the manifest for both AMP-AD and PEC, which helps somewhat, but of course that doesn't mean it is there,....

Exactly. I think I'd need to only update the table if the column for it existed and the column had values in it. There is still the issue that the contributor may have put the wrong grant, typed it wrong, etc.

I just had another thought. What if, instead of adding another column and filling it with the grant value, I added a fourth box at the top next to the % success box. If the grant exists in the files uploaded, I can put the value there. It would still be dependent on the contributor getting it correct, but might be simpler than adding it to the table (the data files and information table are completely separate in dccmonitor).

**Comment by kelshmo** [comment]
That sounds great!
